test/testModuleUsage.js

```diff
@@ -190,6 +192,9 @@ describe('Using ERC-Checker as node-module', function () {
 								done();
 							}
 							else {
+								let meta = metadata;
+								meta.resultHTML = '';
+								debug(meta)
```

Comment:
No need to debug in tests, because it leads to sloppy checks. Make your assertions communicate everything that is needed instead.